pkg/util/util.go

```diff
@@ -1465,7 +1461,7 @@ func GitSubDir(srcPath, destinationPath, subDir string) error {
 
 // gitSubDir handles subDir for git components
 func gitSubDir(srcPath, destinationPath, subDir string, fs filesystem.Filesystem) error {
-	go StartSignalWatcher(func() {
+	go StartSignalWatcher([]os.Signal{syscall.SIGHUP, syscall.SIGINT, syscall.SIGTERM, syscall.SIGQUIT, os.Interrupt}, func() {
```

Comment:
Why aren't we capturing `os.Kill` here? I'm asking because that would help us reuse:
```go
captureSignals := []os.Signal{syscall.SIGHUP, syscall.SIGINT, syscall.SIGTERM, syscall.SIGQUIT, os.Interrupt, os.Kill}
```
defined in `pkg/odo/genericclioptions/runnable.go`.

Reply:
I am not sure why it wasn't used, Mrinal wrote this function, so he must've had his reasons.